cmd/utils/flags.go

```diff
@@ -1580,6 +1597,19 @@ func SetKlayConfig(ctx *cli.Context, stack *node.Node, cfg *cn.Config) {
 		cfg.RPCGasCap = new(big.Int).SetUint64(ctx.GlobalUint64(RPCGlobalGasCap.Name))
 	}
 
+	// Only CNs could set BlockGenerationIntervalFlag and BlockGenerationTimeLimitFlag
+	if ctx.GlobalIsSet(BlockGenerationIntervalFlag.Name) {
+		params.BlockGenerationInterval = ctx.GlobalInt64(BlockGenerationIntervalFlag.Name)
```

Comment:
```suggestion
		params.BlockGenerationInterval = ctx.GlobalInt64(BlockGenerationIntervalFlag.Name)
		params.BlockGenerationTimeLimit = params.BlockGenerationInterval / 4
```
How about setting `BlockGenerationTimeLimit` to 1/4 of `BlockGenerationIntervalFlag` after?

Reply:
@winnie-byun In that case, `BlockGenerationInterval` could be changed unintentionally. I think providing flexibility is better for this kind of experimental feature. Indeed, we don't have a rationale behind `BlockGenerationTimeLimit = BlockGenerationInterval / 4`